python/modem-firmware-1.3+/device_credentials_installer.py

```diff
@@ -648,6 +656,10 @@ def main():
     print(local_style('Writing AWS CA to modem...'))
     if is_gateway:
         modem_ca = aws_ca.replace("\n", "\\n")
+    elif args.coap:
+        modem_ca = coap_ca + aws_ca
+    elif args.prov:
```

Comment:
should a warning be printed to let the user know that `--coap` and `--prov` are mutually exclusive?